lib/mix/lib/mix/compilers/elixir.ex

```diff
@@ -188,8 +189,9 @@ defmodule Mix.Compilers.Elixir do
     Mix.shell.info "Compiled #{source}"
   end
 
-  defp each_long_compilation(source) do
-    Mix.shell.info "Compiling #{source} (it's taking more than 5s)"
+  defp each_long_compilation(source, threshold_in_ms) do
+    threshold_in_s = round(div(threshold_in_ms, 1_000))
```

Comment:
Looks like 1. I'm an idiot and 2. that's what I had in mind initially 😄 Updated!